src/Containers/SessionChoice/styles.ts

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import palette from 'palette';
+
```

Comment:
We should separate library imports from project imports. Consider it and apply it to all files 😅